### DISCUSSION TITLE
xplat app has black screen on android emulator

### DISCUSSION THREAD

**Discussion by johnabbott72** [AvaloniaUI/Avalonia#12284] [General]
I am trying to get my app to run on Android through the emulator in JetBrains and it compiles and installs successfully but when it opens it has a blank screen.  Here are a couple of things I am seeing in the console:

07-21 12:08:26.090 I/NetworkScheduler.Stats( 1888): Task com.google.android.gms/com.google.android.gms.auth.account.be.legacy.AuthCronService started execution. cause:4 exec_start_elapsed_seconds: 598 [CONTEXT service_id=218 ]
07-21 12:08:56.168 W/GLSUser ( 1888): [AppCertManager] IOException while requesting key: 
07-21 12:08:56.168 W/GLSUser ( 1888): java.net.SocketTimeoutException: timeout

07-21 12:10:56.318 E/BufferQueueProducer( 1352): [SurfaceView - com.CompanyName.familyTime/crc643c752086d7a96a8f.MainActivity] cancelBuffer: BufferQueue has no connected producer
07-21 12:10:56.344 D/EGL_emulation( 7144): eglMakeCurrent: 0xa36851e0: ver 3 0 (tinfo 0xa36c1ba0)
07-21 12:10:56.371 I/AVALONIA( 7144): Surface Changed
07-21 12:10:56.407 D/EGL_emulation( 7144): eglMakeCurrent: 0xa36851e0: ver 3 0 (tinfo 0xa36c1ba0)
07-21 12:10:56.455 D/EGL_emulation( 7144): eglMakeCurrent: 0x8ba294c0: ver 3 1 (tinfo 0x8bd54c50)
07-21 12:10:56.661 I/WindowManager( 1710): Destroying surface Surface(name=com.android.systemui/com.android.systemui.recents.RecentsActivity) called by com.android.server.wm.WindowStateAnimator.destroySurface:2016 com.android.server.wm.WindowStateAnimator.destroySurfaceLocked:882 com.android.server.wm.WindowState.destroyOrSaveSurface:2100 com.android.server.wm.WindowManagerService.tryStartExitingAnimation:3024 com.android.server.wm.WindowManagerService.relayoutWindow:2904 com.android.server.wm.Session.relayout:224 android.view.IWindowSession$Stub.onTransact:286 com.android.server.wm.Session.onTransact:145 
07-21 12:10:59.455 W/audio_hw_generic( 1470): Not supplying enough data to HAL, expected position 2732535 , only wrote 2487600
07-21 12:11:00.019 D/EGL_emulation( 1919): eglMakeCurrent: 0x9ef6c6e0: ver 3 1 (tinfo 0xa1ec00e0)

Also, I am running JetBrains on linux - ubuntu and I have no problems with the desktop app.

Any thoughts?

**Comment by johnabbott72** [comment]
Additional items I am seeing in the console:

07-22 09:09:35.351 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Avalonia.Markup.Xaml.dll
07-22 09:09:35.374 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Avalonia.Themes.Fluent.dll
07-22 09:09:35.376 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Material.Icons.Avalonia.dll
07-22 09:09:35.376 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly netstandard.dll
07-22 09:09:35.378 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Avalonia.Controls.DataGrid.dll
07-22 09:09:35.633 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly System.Runtime.CompilerServices.Unsafe.dll
07-22 09:09:35.775 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Avalonia.Dialogs.dll
07-22 09:09:35.795 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Avalonia.Markup.dll
07-22 09:09:35.888 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly Material.Icons.dll
07-22 09:09:36.089 I/ActivityManager( 1710): START u0 {cmp=com.CompanyName.familyTime/crc643c752086d7a96a8f.MainActivity} from uid 10080 on display 0
07-22 09:09:36.095 E/SurfaceFlinger( 1352): ro.sf.lcd_density must be defined as a build property
07-22 09:09:36.102 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly System.Numerics.Vectors.dll
07-22 09:09:36.123 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly System.IO.IsolatedStorage.dll
07-22 09:09:36.287 W/monodroid-assembly(10340): open_from_bundles: failed to load assembly System.Security.Cryptography.dll
07-22 09:09:36.602 I/OpenGLRenderer(10340): Initialized EGL, version 1.4

**Comment by xklake** [comment]
Have this been sorted out? i am having the same issue.

**Reply by timunie** [reply]
check 11.1.o-beta1 pls
